specs/gradle/SOURCES/checksum.sha512

```diff
@@ -0,0 +1 @@
+885961b001fdfe221511f66d81f2962adffa6d3a5fd8148dd20591e8ff2222135216fb67fffa4248c814b4e059097f2b767c7e023b6249fbf36d558bf3674b61  gradle-6.8.2-src.zip
```

Comment:
`checksum.sha512` is not required for the build process. This file automatically generated by `rpmbuilder` while building packages. Also, this spec doesn't have a macro for checking these checksums.